package/MDAnalysis/analysis/dssp/dssp.py

```diff
@@ -0,0 +1,404 @@
+        long_run
+        .results
+        .dssp_ndarray
+        .mean(axis=0) > threshold
+    )
+    print(''.join(persistent_residues)[:20])
+
+Running this code produces ::
+
+    '--EEEE----------HHHH'
+
+
+Analysis classes
+----------------
+
+.. autoclass:: DSSP
+   :members:
+   :inherited-members:
+
+   .. attribute:: results.dssp
+
+       Contains the time series of the DSSP assignment
+       as chars N×m :class:`numpy.ndarray` array with content
+       ``[frame, residue]``, where `structure_type` is one of three characters:
+       ['H', 'E', '-'], representing alpha-helix, sheet and loop, respectively.
+
+   .. attribute:: results.dssp_ndarray
+
+       Contains the one-hot encoding of the time series of the DSSP assignment
+       as chars N×mx3 :class:`numpy.ndarray` array with content
+       ``[frame, residue, encoding]``, where ``encoding`` is a (3,) shape
+       :class:`numpy.ndarray` of booleans with axes representing loop '-',
+       helix 'H' and sheet 'E', consequently.
+
+
+Functions
+---------
+
+.. autofunction:: assign
+.. autofunction:: translate
+"""
+
+from typing import Union
+import numpy as np
+from MDAnalysis import Universe, AtomGroup
+
+from ..base import AnalysisBase
+from ...due import due, Doi
+
+due.cite(
+    Doi("10.1002/bip.360221211"),
+    description="DSSP algorithm description",
+    path="MDAnalysis.analysis.dssp",
+    cite_module=True,
+)
+
+del Doi
+
+
+try:  # pragma: no cover
+    from pydssp.pydssp_numpy import (
+        assign,
+        _get_hydrogen_atom_position,
+    )
+
+    HAS_PYDSSP = True
+
+except ModuleNotFoundError:
+    HAS_PYDSSP = False
+    from .pydssp_numpy import (
+        assign,
+        _get_hydrogen_atom_position,
+    )
+
+
+class DSSP(AnalysisBase):
+    """Assign secondary structure using the DSSP algorithm.
+
+    Analyze a selection containing a protein and assign secondary structure
+    using the Kabsch-Sander algorithm :footcite:p:`Kabsch1983`. Only a subset
+    of secondary structure categories are implemented:
+
+    - 'H' represents a generic helix (α-helix, π-helix or :math:`3_{10}` helix)
+    - 'E' represents 'extended strand', participating in beta-ladder (parallel
+      or antiparallel)
+    - '-' represents unordered part
+
+    The implementation was taken from the pydssp package (v. 0.9.0)
+    https://github.com/ShintaroMinami/PyDSSP by Shintaro Minami.
+
+    .. Warning::
+        For the DSSP to work properly, your atoms (either as ``Universe``
+        or ``AtomGroup``) must have only one hydrogen atom "H" per residue,
+        that corresponds to N-bound hydrogen in a peptide bond (or zero hydrogen
+        atoms, if the residue is proline). It is selected using `hydrogen_atom`
+        parameter. It is designed to hande most of the modern syntaxes,
+        but if you encounter an error or unusual results
+        during your run, please report an issue on MDAnalysi
+        `github <https://github.com/MDAnalysis/mdanalysis/issues>`_.
```

Comment:
Maybe the following is clearer?
```suggestion
    .. Warning::
       For DSSP to work properly, your atoms must represent a protein. The
       hydrogen atom bound to the backbone nitrogen atom is matched by name
       as given by the keyword argument `hydrogen_atom`. There may only be 
       a single backbone nitrogen hydrogen atom per residue; the one exception
       is proline, for which there should not exist any such hydrogens.
       The default value of `hydrogen_atom` should to handle the common naming
       conventions in the PDB and in force fields but if you encounter an error
       or unusual results during your run, try to figure out how to select the
       correct hydrogen atoms and report an issue in the MDAnalysis
       `issue tracker <https://github.com/MDAnalysis/mdanalysis/issues>`_.
```

Reply:
That's more clear indeed.